Dockerfile

```diff
@@ -1,4 +1,4 @@
-# syntax=docker/dockerfile:1.2
+# syntax=docker/dockerfile:1.3
```

Comment:
While this works, I'd prefer to use `1.3.0` like it is done in the documentation.

Reply:
That apparently depends on the docs you look at, e.g. https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#build-mounts-run---mount has `# syntax=docker/dockerfile:1.3`. Also, if it works like for Docker image tags, `1.3` actually means `1.3.x`, so you'll get patch-level updates automatically, which probably is a good thing.

Reply:
> ... so you'll get patch-level updates automatically, which probably is a good thing.

Unless the next release is called 1.3.33-liberty-2 ;)